src/Command/InfectionCommand.php

```diff
@@ -210,52 +219,99 @@ protected function configure(): void
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $adapter = $this->startUp();
```

Comment:
I'm not 100% happy with these method names, but it was the best i could come up with right now, so i'm open to suggestions.

Reply:
While I like the simplification of the main `execute()` method, I would suggest to name these methods as clear as possible, to understand from their names what they really do.

A while ago, I created such image with the mutation process to explain to other people how it really works. Probably it will give us ideas how to name these methods

![infection-flow](https://user-images.githubusercontent.com/3725595/62730498-ba62b500-ba28-11e9-854d-e96fc08e111c.png)

For example, my suggestions:

* `startUp()` - not so descriptive, but at the moment I don't know a better option
* `initialTest()` - I would rename it to `runInitialTestSuite()` or similar. The verb `run` is a key here (IMO)
* `infectedTest()` - let's not use different words. We already have `mutated` like for the process `MutatedProcess`. It can be renamed to `runMutationTesting()`, indicating there is a whole process (set of tests), not just one test
* `check()` - is too common. Probably `checkMetrics()`?